sc/shared/src/main/scala/scalan/staged/Transforming.scala

```diff
@@ -152,7 +148,7 @@ trait Transforming { self: Scalan =>
     protected def mirrorElem(node: Sym): Elem[_] = node.elem
 
     // every mirrorXXX method should return a pair (t + (v -> v1), v1)
-    protected def mirrorVar[A](t: Transformer, rewriter: Rewriter, v: Ref[A]): Transformer = {
+    protected def mirrorVar[A](t: Transformer, v: Ref[A]): Transformer = {
```

Comment:
Do you mean any PRs or plans ?

Reply:
No plans, I just remember they were useful.

Reply:
Then I see no any logical reason to leave unused arguments in functions, in case of need they can be quickly added again, as usages will be in this repo anyway